libs/generic-view/ui/src/lib/predefined/import-contacts/import-contacts-error.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React, { FunctionComponent } from "react"
+import { ButtonAction, IconType } from "generic-view/utils"
+import { ModalButtons, ModalTitleIcon } from "../../interactive/modal"
+import { ButtonSecondary } from "../../buttons/button-secondary"
+import { defineMessages } from "react-intl"
+import { intl } from "Core/__deprecated__/renderer/utils/intl"
+
+const messages = defineMessages({
+  title: {
+    id: "module.genericViews.importContacts.failure.title",
+  },
+  defaultErrorMessage: {
+    id: "module.genericViews.importContacts.failure.defaultErrorMessage",
+  },
+  closeButtonLabel: {
+    id: "module.genericViews.importContacts.failure.closeButtonLabel",
+  },
+})
+
+export interface CustomError {
```

Comment:
CustomError interface is already defined in `libs/generic-view/ui/src/lib/predefined/backup/backup-error.tsx` maybe move it to model lib.